packages/cfd/src/Containers/cfd-password-modal.tsx

```diff
@@ -852,11 +851,12 @@ const CFDPasswordModal = observer(({ form_error, platform }: TCFDPasswordModalPr
 
         return (
             <Localize
-                i18n_default_text='Congratulations, you have successfully created your {{category}} <0>{{platform}}</0> <1>{{type}}</1> account. '
+                i18n_default_text='Congratulations, you have successfully created your {{category}} {{deriv}} <0>{{platform}}</0> <1>{{type}}</1> account. '
```

Comment:
won't be there an extra space if `deriv` is empty string?

Reply:
![image](https://github.com/binary-com/deriv-app/assets/91878582/8bebe667-9c25-4142-92e2-9eb8fd9d9a99)
<img width="370" alt="image" src="https://github.com/binary-com/deriv-app/assets/91878582/b5240786-1057-435a-b96b-dd04c5032879">

Reply:
I do not think so. I attached screenshots for you to check it as well

Reply:
No there wouldn't be extra space.
cc: @shahzaib-deriv

Reply:
Cool. Thanks guys